tests/cpu_bugs/xen_domu_mitigation_test.pm

```diff
@@ -701,22 +701,48 @@ sub install_domu {
               . " --vnc"
               . " --events on_reboot=destroy"
               . " --serial pty", timeout => 5400);
+        if ($status =~ /Domain is still running. Installation may be in progress/) {
```

Comment:
I am not sure if this scope is right and can be used to decide the guest installation is successful.

Reply:
I have observed several installations and so far i see no problemsw with this statement installation process

Reply:
Okay, i have no problem if you already verified